tests/unit/test_merge_split.cairo

```diff
@@ -0,0 +1,232 @@
+%lang starknet
+
+from starkware.cairo.common.bool import TRUE, FALSE
+from starkware.cairo.common.uint256 import Uint256
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+
+from openzeppelin.introspection.erc165.library import ERC165
+from openzeppelin.security.safemath.library import SafeUint256
+from openzeppelin.token.erc721.enumerable.library import ERC721Enumerable
+from openzeppelin.token.erc721.library import ERC721
+
+from carbonable.erc3525.extensions.slotenumerable.library import ERC3525SlotEnumerable
+from carbonable.erc3525.library import ERC3525
+from carbonable.erc3525.presets.ERC3525Full import split, merge
+from carbonable.erc3525.utils.constants.library import IERC3525_RECEIVER_ID
+
+from tests.unit.library import assert_that
+
+const NAME = 'Carbonable Project';
+const SYMBOL = 'CP';
+const DECIMALS = 18;
+const ADMIN = 'admin';
+const USER1 = 'user1';
+const USER2 = 'user2';
+const USER3 = 'user3';
+const TOKN1 = 1;
+const TOKN2 = 2;
+const INVALID_TOKEN = 666;
+const SLOT1 = 'slot1';
+const SLOT2 = 'slot2';
+
+@external
+func __setup__{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() {
```

Comment:
Add boundary tests :
- try to split and merge with only 1 token_id
- try to split and merge with only 2 token_id